lib/rules/no-test-id-queries.ts

```diff
@@ -0,0 +1,46 @@
+import { TSESTree } from '@typescript-eslint/utils';
+
+import { createTestingLibraryRule } from '../create-testing-library-rule';
+
+export const RULE_NAME = 'no-test-id-queries';
+export type MessageIds = 'noTestIdQueries';
+type Options = [];
+
+const QUERIES_REGEX = /^(get|query|getAll|queryAll|find|findAll)ByTestId$/;
```

Comment:
suggestion (blocking): Please build this regex using the `ALL_QUERIES_VARIANTS` from the utils file.

Reply:
Good idea, done